### ISSUE TITLE
How can I use css with this Markup?!

### ISSUE THREAD

**Issue by marwajomaa** [milesj/interweave#192]
I have HTML content and CSS from backed and I render the HTML using 
interweave but I need to add CSS, can I do that?!

**Comment by milesj** [comment]
@marwajomaa How does css work? Is it in a `style` element or something?

**Comment by marwajomaa** [comment]
It's dynamic css from backend also I have css file need to add both, this
is the html and css from backend response

**Comment by marwajomaa** [comment]
It would be great if there's a way to do that

**Comment by milesj** [comment]
@marwajomaa Can you provide an example of what the response looks like?

**Comment by marwajomaa** [comment]
Hi @milesj I receive a JSON object from the backend and it contains HTML and CSS shown in the previous comment, then I use interweave npm to show HTML in the browser but no style I need to add the CSS as well, it's coming from the same JSON object as dynamic style also I have style file I want to add to that HTML  markup 

this how html from backend in the console
<img width="288" alt="Capture" src="https://user-images.githubusercontent.com/34250163/130231872-c8de163d-c78d-4fef-81c8-2931ebdc7d30.PNG">

so this is the css as styles and HTML as body from the backend

<img width="747" alt="Capture" src="https://user-images.githubusercontent.com/34250163/130232204-30c7b64c-425e-40f2-970b-fe9430d9142f.PNG">

and thats how I use markup to show html in the browser
<img width="221" alt="Capture" src="https://user-images.githubusercontent.com/34250163/130231328-0d413b95-55b6-4d47-ad1d-1854e90a34fa.PNG">

the page only contains plain HTML with zero style and images also not showing, any fix for that?!

is that clear enough or you more explain let me know, please. thanks

**Comment by milesj** [comment]
@marwajomaa So theres no way to do this in interweave directly, but you could so something like this: https://stackoverflow.com/a/524721

```
const head = document.head || document.getElementsByTagName('head')[0],
    style = document.createElement('style');

style.appendChild(document.createTextNode(res.styles));
head.appendChild(style);
```

**Comment by marwajomaa** [comment]
Thanks @milesj,  so is that supposed to work with using Markup for HTML?

and how can I attach a CSS file also to that HTML is there any way? 

the last thing is about images not rendered from html how can show them. this is how they look in the console
<img width="457" alt="Capture" src="https://user-images.githubusercontent.com/34250163/130274733-92cd7e80-5f04-4aca-9a90-92a49c3505b2.PNG">

I tried to add an img folder next to the HTML page (using next.js)  and inside img folder, I added team1.jpg image but still not showing
